config_defaults.yml

```diff
@@ -27,9 +27,34 @@ Gemfile:
         version: '~>0.10.5'
       - gem: pry
     ':system_tests':
-      - gem: beaker-rspec
       - gem: beaker
+        from_env: 'BEAKER_VERSION'
+        version: '~> 2.20'
+        condition: "supports_windows"
+      - gem: beaker
+        from_env: 'BEAKER_VERSION'
+        condition: "Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.3.0') and ! supports_windows"
+      - gem: beaker
+        from_env: 'BEAKER_VERSION'
+        version: '< 3'
+        condition: "Gem::Version.new(RUBY_VERSION) < Gem::Version.new('2.3.0') and ! supports_windows"
+      # beaker 3 requires this for PE installs
+      - gem: beaker-pe
+        condition: "Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.3.0')"
+      - gem: beaker-rspec
+        from_env: 'BEAKER_RSPEC_VERSION'
+        # newer version required to avoid BKR-537
+        version: '>= 3.4'
+        condition: "! supports_windows"
+      - gem: beaker-rspec
+        from_env: 'BEAKER_RSPEC_VERSION'
+        # newer version required to avoid BKR-537
+        version: '~> 5.1'
+        condition: "supports_windows"
```

Comment:
I agree in general - this is definitely the goal to bring the gems to the same version cross platform.